Week1/homework/app.js

```diff
@@ -126,13 +124,12 @@ function createBookList(bookTitles, bookCovers) {
 
     let image = document.createElement("img");
     li.appendChild(image);
-    image.src = getBookDetails(i, bookCovers);
-    image.setAttribute("width", 150);
+    image.src = getBookDetails(key, bookCovers);
+    image.setAttribute("width", 50);
   }
-
+  return ul;
   function getBookDetails(src, bookCovers) {
```

Comment:
Never put anything after `return` as you saw in the lesson it would be tricky to have a control over it sometimes.

this function is a utility function we may move it outside of the `createBookList` because it's not doing anything with it.